image/templates/helm/stackrox-central/values.yaml.htpl

```diff
@@ -193,7 +184,68 @@
 #      # Specify a custom hostname if desired, otherwise accept the default from OpenShift.
 #      host: null
 #
+[< if .FeatureFlags.ROX_POSTGRES_DATASTORE >]
+#  db:
+#    enabled: false
+#
+#    # Preexisting signifies that a Postgres wire-compatible database has already been deployed and a Central DB pod
+#    # does not need to be deployed
+#    preexisting: false
```

Comment:
Before getting to this line, I had a misconception about what `preexisting` meant. It's confusing because `helm` values aren't used for newly deploying only, but also for upgrading, and in case of anything but the first upgrade, the DB will also be pre-existing, yet still is Helm-managed. Maybe call this `external`?

Reply:
External works